lib/src/inline_parser.dart

```diff
@@ -972,7 +1016,7 @@ class LinkSyntax extends TagSyntax {
   @override
   Node? close(
       InlineParser parser, covariant SimpleDelimiter opener, Delimiter? closer,
-      {required List<Node> Function() getChildren}) {
+      {String? tag, required List<Node> Function() getChildren}) {
```

Comment:
Style nit: please add a trailing comma here, so that each parameter appears on its own line.

Reply:
Fixed